<?xml version="1.0" encoding="UTF-8"?>
<!--
  - Copyright (c) 2024. aaron.
  -
  - This program is under the GPL-3.0 license.
  - if you have not received it or the program has a bug, please let me know:
  - <communicate_aaron@outlook.com>.
  -->

<ui version="4.0">
    <class>Widget</class>
    <widget class="QWidget" name="Widget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>622</width>
                <height>413</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>元对象系统功能示例</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QGridLayout" name="gridLayout">
                    <item row="0" column="2">
                        <widget class="QPushButton" name="pushbutton_boy_grow_up">
                            <property name="text">
                                <string>男生长大了一岁</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QSpinBox" name="spinbox_girl">
                            <property name="minimum">
                                <number>1</number>
                            </property>
                            <property name="maximum">
                                <number>120</number>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QSpinBox" name="spinbox_boy">
                            <property name="minimum">
                                <number>1</number>
                            </property>
                            <property name="maximum">
                                <number>120</number>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="3">
                        <widget class="QPushButton" name="pushbutton_display_metainfo">
                            <property name="text">
                                <string>元对象信息</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="2">
                        <widget class="QPushButton" name="pushbutton_girl_grow_up">
                            <property name="text">
                                <string>女生长大了一岁</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="0">
                        <widget class="QLabel" name="label_boy">
                            <property name="text">
                                <string>设置男生年龄</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="label_girl">
                            <property name="text">
                                <string>设置女士年龄</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="3">
                        <widget class="QPushButton" name="pushbutton_clean">
                            <property name="text">
                                <string>清空文本</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="Line" name="line">
                    <property name="orientation">
                        <enum>Qt::Orientation::Horizontal</enum>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QPlainTextEdit" name="plaintext_edit"/>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
